core/src/test/scala/unit/kafka/server/ReplicaManagerTest.scala

```diff
@@ -1084,40 +1090,46 @@ class ReplicaManagerTest {
       topicPartition, leaderEpoch + leaderEpochIncrement, followerBrokerId,
       leaderBrokerId, countDownLatch, expectTruncation = true, topicId = Some(topicId))
 
-    val brokerList = Seq[Integer](0, 1).asJava
+    try {
```

Comment:
Most of the changes here are indentation changes  to add `try { ... } finally { replicaManager.shutdown() }`